tests/test_harvest_start_date_honoring.py

```diff
@@ -1,16 +1,15 @@
-"""
-Test that the start_date is respected for some streams
-"""
-from tap_tester import runner
-
+from tap_tester import runner, menagerie, LOGGER
 from harvest_api import *
 from base import BaseTapTest
 
 class TestStartDateHonoring(BaseTapTest):
-    
+    """
+    Test that the start_date is respected for some streams
+    """
+
     @classmethod
     def setUpClass(cls):
-        logging.info("Start Setup")
+        LOGGER.info("Start Setup")
         # Track what was created to delete in teardown
         cls._teardown_delete = {"contacts": [], "projects": [], "project_tasks": [],
```

Comment:
Is it similar set like base.py?

Reply:
No, it is different set from the `base.py`. We do not have such set in the `base.py`